WORKSPACE

```diff
@@ -4,41 +4,33 @@ load("//bazel:repositories.bzl", "api_dependencies")
 
 api_dependencies()
 
+# TODO(htuch): This can switch back to an http_archive at the next release (>
```

Comment:
optional: you can use `http_archive` with Github archives at any commit ID.

`https://github.com/google/protobuf/archive/c4f59dcc5c13debc572154c8f636b8a9361aacde.tar.gz`

Reply:
Yeah, this is better, it's easier for folks with proxies, since Bazel uses some Java git library that is hard to configure.